lib/routes/home/widgets/payments_list/dialog/payment_details_dialog_expiration.dart

```diff
@@ -39,7 +44,21 @@ class PaymentDetailsDialogExpiration extends StatelessWidget {
               group: labelAutoSizeGroup,
             ),
           ),
-          // TODO: Add pendingExpirationTimestamp information once implemented
+          Expanded(
+            child: SingleChildScrollView(
+              scrollDirection: Axis.horizontal,
+              reverse: true,
+              padding: const EdgeInsets.only(left: 8.0),
+              child: AutoSizeText(
+                pendingExpirationTime != null
+                    ? BreezDateUtils.formatYearMonthDayHourMinute(pendingExpirationTime)
+                    : "",
+                style: themeData.primaryTextTheme.displaySmall,
+                textAlign: TextAlign.right,
+                maxLines: 1,
+              ),
+            ),
+          )
         ],
```

Comment:
Maybe it's better to use a helper method for pending expiration time so the logic is more compact.

```
AutoSizeText(
    _getExpirationTime,
    ...
...
_getExpirationTime() {
    final blockHeight = context.read<AccountBloc>().state.blockheight;
    final pendingExpirationBlock = paymentMinutiae.pendingExpirationBlock;
    if(pendingExpirationBlock != null) {
        final pendingExpirationTime = BreezDateUtils.blockDiffToDate(
            blockHeight: blockHeight,
            expiryBlock: pendingExpirationBlock,
        );
        return BreezDateUtils.formatYearMonthDayHourMinute(pendingExpirationTime);
    }
   return "";
}
```

Reply:
Removed this logic and added a `pendingExpirationTime` to the PaymentMinutiae also.